### PR TITLE
Added helmet config to Atlas #39

### LINKED ISSUE #39: Add security headers

### PR DESCRIPTION
**Pull request by Haluk-GeniusYield**
Closes #39

### CODE REVIEW

**Review by Dacomis — DISMISSED** [review]

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -30,10 +31,31 @@ function App({ Component, pageProps }: AppProps) {
...
         }}
       />
-      <ThemeProvider theme={theme}>
-        <Component {...pageProps} />
-      </ThemeProvider>
     </>
+    <ThemeProvider theme={theme}>
+      <Helmet>
+        <meta charSet="utf-8" />
+        <title>Your Website Title</title>
```
Comment (sourabhxyz) [inline src/pages/_app.tsx:38]: I would like to mention that we have configured title/description here: https://github.com/geniusyield/atlas-docs/blob/main/theme.config.tsx#L16, want to make sure that you are aware of it.

**Review by sourabhxyz — CHANGES_REQUESTED** [review]

File: src/pages/_app.tsx (hunk 1 above)
Comment (Haluk-GeniusYield) [reply src/pages/_app.tsx]: Updated

**Review by sourabhxyz — CHANGES_REQUESTED** [review]
It seems that `yarn run dev` is failing.

**Comment by Haluk-GeniusYield** [comment]
> It seems that `yarn run dev` is failing.

Should be fixed now

**Review by Dacomis — DISMISSED** [review]

**Review by sourabhxyz — APPROVED** [review]

**Review by sourabhxyz — CHANGES_REQUESTED** [review]
@Haluk-GeniusYield In the last chapter of documentation, we run queries against our locally ran server, the given restrictions introduced in this PR disables it. Is it possible to whitelist server's url (`localhost:8081`)?
image

**Comment by Haluk-GeniusYield** [comment]
> @Haluk-GeniusYield In the last chapter of documentation, we run queries against our locally ran server, the given restrictions introduced in this PR disables it. Is it possible to whitelist server's url (`localhost:8081`)? image

Doing this for production is dangerous. We need setup a development environment and allow localhost in there. I will take a look later today.

**Comment by Haluk-GeniusYield** [comment]
@Dacomis @sourabhxyz added a check to disable CSP during local development. Please have a look and let me know what you think

**Review by sourabhxyz — APPROVED** [review]

**Pull request merged**